doc/api/intl.md

```diff
@@ -138,23 +108,20 @@ the intended ICU version, and `b` or `l` indicates the system's endianness.
 Check ["ICU Data"][] article in the ICU User Guide for other supported formats
 and more details on ICU data in general.
 
-The [full-icu][] npm module can greatly simplify ICU data installation by
-detecting the ICU version of the running `node` executable and downloading the
-appropriate data file. After installing the module through `npm i full-icu`,
-the data file will be available at `./node_modules/full-icu`. This path can be
-then passed either to `NODE_ICU_DATA` or `--icu-data-dir` as shown above to
-enable full `Intl` support.
-
 ### Embed the entire ICU (`full-icu`)
 
 This option makes the resulting binary link against ICU statically and include
 a full set of ICU data. A binary created this way has no further external
-dependencies and supports all locales, but might be rather large. See
-[BUILDING.md][BUILDING.md#full-icu] on how to compile a binary using this mode.
+dependencies and supports all locales. This is the default.
+
+### `small-icu`
+
+Support for `small-icu` (English only) has been removed. See ICU’s
+documentation for information on how to customize your ICU build.
```

Comment:
should link to: http://userguide.icu-project.org/packaging#TOC-Reduce-ICU-Data-used